vsintegration/Utils/LanguageServiceProfiling/Program.fs

```diff
@@ -0,0 +1,69 @@
+﻿open Microsoft.FSharp.Compiler
+open Microsoft.FSharp.Compiler.SourceCodeServices
+open System
+open System.IO
+open LanguageServiceProfiling
+open System.Diagnostics
+open System.Collections.Generic
+
+[<EntryPoint>]
+let main argv = 
+    let rootDir = argv.[0]
+    let options = Options.FCS rootDir
+    let checker = FSharpChecker.Create()
+    let waste = new ResizeArray<int array>()
+    let checkProject() =
+        printfn "ParseAndCheckProject(FCS)..."
+        let sw = Stopwatch.StartNew()
+        let result = checker.ParseAndCheckProject(options) |> Async.RunSynchronously
+        if result.HasCriticalErrors then
+            printfn "Finished with ERRORS: %+A" result.Errors
+        else printfn "Finished successfully in %O" sw.Elapsed
+        ()
+
+    let checkFile (relativePath: string) (fileVersion: int) =
+        printfn "ParseAndCheckFileInProject(%s)..." relativePath
+        let filePath = Path.Combine(rootDir, relativePath)
+        let sw = Stopwatch.StartNew()
+        let _, answer = checker.ParseAndCheckFileInProject(filePath, fileVersion, File.ReadAllText filePath, options) |> Async.RunSynchronously
```

Comment:
Would be great to add some options to  perform some GetDeclarations, GetToolTip calls and the FCS calls needed to support find-all-references